target/msm8226/init.c

```diff
@@ -226,35 +228,56 @@ void target_crypto_init_params()
 
 void target_sdc_init()
 {
-	struct mmc_config_data config = {0};
+	struct mmc_config_data config;
+	struct mmc_config_data sd_config;
 
-	/*
-	 * Set drive strength & pull ctrl for emmc
-	 */
-	set_sdc_power_ctrl();
-
-	config.bus_width = DATA_BUS_WIDTH_8BIT;
+#ifndef FORCE_SDCARD
+	/* Try slot 1*/
+	config.bus_width    = DATA_BUS_WIDTH_8BIT;
+	config.slot         = 1;
 	config.max_clk_rate = MMC_CLK_200MHZ;
-
-	/* Trying Slot 1*/
-	config.slot = 1;
-	config.sdhc_base = mmc_sdhci_base[config.slot - 1];
-	config.pwrctl_base = mmc_pwrctl_base[config.slot - 1];
-	config.pwr_irq     = mmc_sdc_pwrctl_irq[config.slot - 1];
+	config.sdhc_base    = mmc_sdhci_base[config.slot - 1];
+	config.pwrctl_base  = mmc_pwrctl_base[config.slot - 1];
+	config.pwr_irq      = mmc_sdc_pwrctl_irq[config.slot - 1];
```

Comment:
Would prefer to avoid the "code churn" here (just keep it badly aligned since this is qcom code), a minimal diff will look easier to understand